tasks/libs/ninja_syntax.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/python
```

Comment:
Could you add a comment in this file to say that this was extracted from https://github.com/ninja-build/ninja, with the commit / version used?

Reply:
Done!